pulpcore/cli/core/context.py

```diff
@@ -111,6 +112,21 @@ def find(self, **kwargs: Any) -> Any:
         return search_result[0]
 
 
+class PulpGroupUserContext(PulpEntityContext):
+    ENTITY = "group user"
+    HREF = "auth_auth_groups_user_href"
```

Comment:
Reminder to self:
This needs to be changed into a workaround for pulpcore < 3.10

Reply:
done